src/Tools/MicroBuild/publish-assets.ps1

```diff
@@ -35,15 +35,11 @@ try
     # We need to remove 'refs/heads/' from the beginning of the string
     $branchName = $branchName -Replace "^refs/heads/"
     
-    # We also need to replace all instances of '/' with '_'
-    $branchName = $branchName.Replace("/", "_")
```

Comment:
Hmm.  Trying to remember why this line existed.  Wonder if at one point this ended up going into the package name.  

Can't think of why now though ...  Lets proceed and see if anyting breaks.